app/javascript/packs/application.js

```diff
@@ -26,7 +26,7 @@ require("channels")
 import "bootstrap";
 
 // Internal imports, e.g:
-// import { initSelect2 } from '../components/init_select2';
+import { functName } from './user';
```

Comment:
What's this for?

Reply:
I created a users folder in JS in case I need some JS for the profile. If I do write some JS I would replace functName in application with the actual name of the JS function

Reply:
But since the function doesn’t exist it will break the javasript atm.. so let’s comment it out

Reply:
Ok done